openlibrary/macros/LoanStatus.html

```diff
@@ -115,11 +115,8 @@
   $if secondary_action:
     $:macros.BookPreview(ocaid, linkback=not no_index)
 
-$elif ('eligibility' in doc) or (check_sponsorship and not ocaid and not availability.get('is_lendable')):
-  $ sponsorship = doc.get('eligibility') if 'eligibility' in doc else qualifies_for_sponsorship(doc, donate_only=True)
-  $if sponsorship.get('is_eligible'):
-    <a href="$(sponsorship['sponsor_url'])"
-       class="cta-btn cta-btn--sponsor dialog--open"
+$elif ('eligibility' in doc) or (check_sponsorship and not ocaid and qualifies_for_sponsorship(doc, donate_only=True).get('is_eligible')):
```

Comment:
This likely needs to be updated to `'eligibility' in doc and not ocaid` and we may also want to consider doing this if and only if `not secondary_action`